CatAnalyzer/test/TTdilepDraw/TTdilepDraw.py

```diff
@@ -23,118 +34,140 @@ def hist_maker(name, title, bin_set, x_name, y_name, tr, br, cut):
               'WW_TuneCUETP8M1_13TeV',
               'WZ_TuneCUETP8M1_13TeV',
               'DYJetsToLL_M-50']
-mcval_l = [831.8, 61526.7, 35.6, 35.6, 31.8, 65.9, 118.7, 6025.2]
-color_l = [2, 0, 3, 5, 7, 7, 7, 4]
-
 rdfilelist = ['MuonEG']
 
+mcval_l = [815.96, 61526.7, 35.6, 35.6, 15.4, 110.8, 66.1, 6025.2]
+nevent_l = [19899500, 24151270, 1000000, 995600, 996168, 994416, 991232, 28825132]
```

Comment:
These info can be retrieved from the samples.json file. do you want me to modify these lines to use it?